### DISCUSSION TITLE
Deepin V23安装SecureCRT 9.3.2

### DISCUSSION THREAD

**Discussion by Linux-qitong** [linuxdeepin/developer-center#4616] [Troubleshooting | 故障排查]
原贴:https://bbs.deepin.org/phone/zh/post/257549
已购买了SecureCRT,装的是SecureCRT当前最新版:9.3.2，但是却运行不起来

难道只能编译libstdc++吗?

报错如下
$ SecureFX
SecureFX: /lib/x86_64-linux-gnu/libstdc++.so.6: version GLIBCXX_3.4.30' not found (required by SecureFX) SecureFX: /lib/x86_64-linux-gnu/libstdc++.so.6: version GLIBCXX_3.4.30' not found (required by /usr/bin/../lib/scrt-sfx/libClientConfigUiQt.so)

$ SecureCRT
SecureCRT: /lib/x86_64-linux-gnu/libstdc++.so.6: version GLIBCXX_3.4.30' not found (required by SecureCRT) SecureCRT: /lib/x86_64-linux-gnu/libstdc++.so.6: version GLIBCXX_3.4.30' not found (required by /usr/bin/../lib/scrt-sfx/libClientConfigUiQt.so)

系统信息
$ hostnamectl
Static hostname: wedo-PC
Icon name: computer-laptop
Chassis: laptop 💻
Machine ID: 691a0de559c149a5b75a843810b236e8
Boot ID: 00a9bf50f0e144fdaeb2906aab982ca9
Operating System: Deepin 23
Kernel: Linux 6.1.11-amd64-desktop-hwe
Architecture: x86-64
Hardware Vendor: Lenovo
Hardware Model: ThinkPad T14 Gen 1

**Comment by BLumia** [comment]
商业软件的支持不是应该联系商业软件的开发商（指 SecureCRT 的开发商）吗？

不过运行的话，可以考虑从 distrobox 环境里运行。比如原本支持的环境是 ubuntu，那就新建一个 ubuntu 的 box 然后在里面跑 SecureCRT。
